### ISSUE TITLE
Native Ad wrongly shown on iOS

### ISSUE THREAD

**Issue by ecorradini** [dreamsoftin/facebook_audience_network#95]
Native ads are wrongly shown on iOS app.
![IMG_CE87F6A1B914-1](https://user-images.githubusercontent.com/13837299/108223657-513e2080-713a-11eb-9404-3d95649e4d57.jpeg)
However, the same Native Ad works on Android.
![IMG_1246](https://user-images.githubusercontent.com/13837299/108223808-7af74780-713a-11eb-9114-230e0e9762b0.png)
This is the code:
`FacebookNativeAd(
        placementId: Platform.isIOS ? "iOS_id" : "Android_id",
        adType: NativeAdType.NATIVE_AD,
        backgroundColor: Theme.of(context).backgroundColor,
        titleColor: Colors.white,
        descriptionColor: Colors.white,
        buttonColor: Theme.of(context).accentColor,
        buttonTitleColor: Colors.white,
        buttonBorderColor: Theme.of(context).accentColor,
        keepAlive: true,
        keepExpandedWhileLoading: false,
    );`

**Comment by ecorradini** [comment]
Update:
Changing to NATIVE_AD_HORIZONTAL for iOS prevents the ad to be shown like the screenshot above. However, the first time the ad is loaded a blank space is left instead of the ad.
I've seen that this behavior is caused by `keepExpandedWhileLoading: false`. Settings this to `true` the ad is shown correctly the first time.
However, if the ad is not loaded it will leave the blank space.

**Comment by SachinGanesh** [comment]
Yes. keepExpandedWhileLoading is used to prevent the blank space from showing
